src/pudl/transform/ferc1.py

```diff
@@ -790,11 +789,27 @@ def check_table_calculations(
             f"{table_name}: has {len(off_df)} ({off_ratio:.02%}) records that don't "
             "calculate exactly."
         )
+
+    if off_ratio > 0 and off_ratio <= params.calculation_tolerance:
+        logger.info(
+            "Creating correction records to make calculations match reported values."
+        )
+        corrections = off_df.copy()
+        corrections[params.column_to_check] = (
+            corrections[params.column_to_check] - corrections["calculated_amount"]
+        )
```

Comment:
```suggestion
        corrections[params.column_to_check] = (
            corrections[params.column_to_check].fillna(0) - corrections["calculated_amount"]
        )
```

we should probs fillna so we get the full cacled amount if the value is null